### ISSUE TITLE
TypeError when load option config

### ISSUE THREAD

**Issue by joe820912boy** [DCsunset/pandoc-include#57]
> Traceback (most recent call last):
>   File "/home/jacky/.local/bin/pandoc-include", line 8, in <module>
>     sys.exit(main())
>   File "/home/jacky/.local/lib/python3.8/site-packages/pandoc_include/main.py", line 429, in main
>     return pf.run_filter(action, doc=doc)
>   File "/home/jacky/.local/lib/python3.8/site-packages/panflute/io.py", line 227, in run_filter
>     return run_filters([action], *args, **kwargs)
>   File "/home/jacky/.local/lib/python3.8/site-packages/panflute/io.py", line 208, in run_filters
>     doc = doc.walk(action, doc=doc, stop_if=stop_if)
>   File "/home/jacky/.local/lib/python3.8/site-packages/panflute/base.py", line 264, in walk
>     child = child.walk(action, doc, stop_if)
>   File "/home/jacky/.local/lib/python3.8/site-packages/panflute/base.py", line 272, in walk
>     altered = action(self, doc)
>   File "/home/jacky/.local/lib/python3.8/site-packages/pandoc_include/main.py", line 254, in action
>     options = parseOptions(doc)
>   File "/home/jacky/.local/lib/python3.8/site-packages/pandoc_include/config.py", line 91, in parseOptions
>     options = defaultOptions | json.load(f)
> TypeError: unsupported operand type(s) for |: 'dict' and 'dict'
> Error running filter pandoc-include:
> Filter returned error status 1

I build with pandoc-inlcude and it occurs type error to options config

and my python3 version is Python 3.8.10

**Comment by DCsunset** [comment]
Please post the command you use and a minimal document to reproduce the problem. Then it would be easier to locate the error.

**Comment by joe820912boy** [comment]
@DCsunset 
```
pandoc --toc --filter ~/.local/bin/pandoc-include --template eisvogel --number-sections --highlight-style tango main.md -o test.pdf
```

and the document i used only empty .md file and it also give the error

**Comment by joe820912boy** [comment]
In my .temp.pandoc.include it look likes.

`{"include-entry": {"path": ".", "entered": true}, "current-path": ".", "include-resources": ".", "process-path": "/path/to/md/folder", "include-order": "natural", "rewrite-path": true, "pandoc-options": ["--filter=pandoc-include"]}
`

**Comment by DCsunset** [comment]
This seems to be an issue with Python 3.8. I made a fix in the master branch. Can you test it again with your current Python version?

**Comment by joe820912boy** [comment]
@DCsunset OK, i will check and give the result

**Comment by joe820912boy** [comment]
This master version is worked for me now!!!

Thanks.

**Comment by DCsunset** [comment]
This fix is included in v1.4.1. Closing it as completed.
